net.peercoin.peercoin-qt.json

```diff
@@ -0,0 +1,71 @@
+{
+  "id": "net.peercoin.peercoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "19.08",
+  "command": "peercoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "peercoin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/peercoin-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/peercoin/peercoin/releases/download/v0.10.4ppc/peercoin-0.10.4-x86_64-linux-gnu.tar.gz",
```

Comment:
it's MIT licensed, why not building from source instead?

Reply:
Figured this is far easier, less stuff to handle.

Reply:
Also there is an aarch64 binary, so why limiting to x86_64.

Reply:
I can add both arm and aarch64 but I doubt it will get many users?

Reply:
it sure won't get any if there is no package.